packages/preview-middleware-client/src/utils/version.ts

```diff
@@ -50,7 +50,7 @@ export async function getUi5Version() {
 export function isLowerThanMinimalUi5Version(
     ui5VersionInfo: Ui5VersionInfo,
     minUi5VersionInfo: Ui5VersionInfo = minVersionInfo
-): boolean {
+) {
```

Comment:
Why the return type was removed from here?

Reply:
Because TypeScript infers the return type anyway. I thought this is a general TS feature and not IDE specific. At least in my IDE `:boolean` is superfluous. If that assumption proves wrong I can of course add it again

Reply:
Added missing return type